config/gemini_retry.py

```
@@ -6,20 +6,52 @@
+            class TooManyRequests(Exception):
+                """429 Too Many Requests 대체 클래스"""
+                pass
+            
+            class ResourceExhausted(Exception):
+                """Resource Exhausted 대체 클래스"""
+                pass
+            
+            class DeadlineExceeded(Exception):
+                """Deadline Exceeded 대체 클래스"""
+                pass
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)